logs.py

```diff
@@ -1,9 +1,35 @@
+from pubnub import Pubnub
+
 from .base_request import BaseRequest
 from .resources import Message
+from .models.config import Config
+
+class Logs(object):
+	def __init__(self):
+		self.config = Config()
+		
+
+	def subscribe(self, uuid, callback, error):
+		pubnub_key = self.config.get_pubnub_keys()
+		if not hasattr(self, 'pubnub'):
+			self.pubnub = Pubnub(publish_key=pubnub_key['publish_key'], subscribe_key=pubnub_key['subscribe_key'])
+		channel = self.get_channel(uuid)
+		self.pubnub.subscribe(channels=channel, callback=callback, error=error)
+
+	def history(self, uuid, callback, error):
+		pubnub_key = self.config.get_pubnub_keys()
+		if not hasattr(self, 'pubnub'):
```

Comment:
BTW is `Pubnub` constructor an expensive operation? if not it can always be called from `__init__`